component/pyroscope/ebpf/ebpf_placeholder.go

```diff
@@ -0,0 +1,3 @@
+//go:build !linux
+
+package ebpf
```

Comment:
Can you register a dummy implementation of the component here which logs a notice that pyroscope.ebpf is no-op on non-Linux platforms similarly to what we do in static mode for the [node_exporter integration](https://github.com/grafana/agent/blob/474fe21362810889f7c55206483613a61ccdb874/pkg/integrations/node_exporter/node_exporter_windows.go#L18-L21)?

Reply:
done